teps/0100-embedded-taskruns-and-runs-status-in-pipelineruns.md

```diff
@@ -171,12 +195,402 @@ In addition to grouping related resources, Tekton Results provides long term sto
 of `PipelineRuns`, `TaskRuns` and `Runs` away from the runtime storage in etcd.
 Read more in [TEP-0021][tep-0021].
 
-## Open Questions
+## Proposal
 
-* [Qn #1][qn-1]: What fields will no longer be available? What do we lose from the optimizations?
-  We will discuss this in the proposal.
-* [Qn #2][qn-1]: What does an example status look like?
-  We will provide examples in the proposal.
+The `PipelineRunStatus` should contain:
+* the names of its `TaskRuns` and `Runs`.
+* the overall status (`ConditionSucceeded`) of its `TaskRuns` and `Runs`.
+* the names of the `PipelineTasks` from which the `TaskRuns` and `Runs` were executed.
+
+Users and tools can find the complete status of `TaskRuns` and `Runs` in the cluster.
+Moreover, users and tools can rely on [Tekton Results](#tekton-results-api) and
+[Owner References and Labels](#owner-references-and-labels) to identify related objects.
```

Comment:
Can we please add an example of how to retrieve `taskRun` and `run` status with and without tekton results?

Reply:
good idea, added these examples in the example section of the proposal, thanks!